pkg/lib/cockpit-components-file-autocomplete.jsx

```diff
@@ -84,9 +84,9 @@ export class FileAutoComplete extends React.Component {
     }
 
     onCreateOption(newValue) {
-        this.setState({
-            displayFiles: [...this.state.displayFiles, { type: "file", path: newValue }]
-        });
+        this.setState(prevState => ({
+            displayFiles: [...prevState.displayFiles, { type: "file", path: newValue }]
+        }));
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18387-20230222-094250-b66008dd-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)